tests/test-sessions/test-infinispan-sessions/src/test/java/org/eclipse/jetty/server/session/InfinispanTestSupport.java

```diff
@@ -25,8 +25,6 @@
 import java.util.Properties;
 
 import org.eclipse.jetty.toolchain.test.FS;
-import org.eclipse.jetty.toolchain.test.jupiter.WorkDir;
-import org.eclipse.jetty.toolchain.test.jupiter.WorkDirExtension;
 import org.eclipse.jetty.util.IO;
```

Comment:
Not a fan of making changes to totally unrelated code ... but as they're unused imports I guess its ok.

Reply:
This looks like changes from a merge.

Reply:
@joakime nope I was just looking at this class and removed unused imports